mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -1666,6 +1667,7 @@ void getMetaDataShouldReturnDefaultValueWhenNoReadRight() throws Exception {
     @Nested
     public class BasicFeaturesTests {
 
+        @Disabled
         @Test
         void renameMailboxShouldReturnAllRenamedResultsIncludeChildren() throws MailboxException {
```

Comment:
I don't know why `rename` method is hanging for this test case. (hanging, not throw exception)(org.apache.james.mailbox.store.StoreMailboxManager#doRenameMailbox)

Some test `rename` work, and some test `rename` hanging. 
I thought about `MailboxPathLocker`, and I tried to `NoMailboxPathLocker` as a monkey doing. But it did not work. 

Somebody can help me debug it?

Reply:
the test case was marked ` @Disabled` is the hanging test

Reply:
Well, main thread await the reactor operation.  Nothing else is currently done by the JVM.

I more suspect a deadlock on PG side.

Reply:
I think I did get it...

Reply:
Can you explain what happened?